api/.php_cs.dist

```diff
@@ -0,0 +1,37 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->exclude(['bin', 'config', 'docker', 'helm', 'var', 'vendor'])
```

Comment:
Only `var` is necessary here, I think.

`bin` and `config` should not be excluded.

`vendor` is already excluded by default: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.15.0/src/Finder.php#L33

The rest don't contain *.php files, I suppose?